doc/docs/usage/tutorial-fleet-management.md

```diff
@@ -0,0 +1,320 @@
+
+```shell
+podman run -d -p 1883:1883 docker.io/eclipse-mosquitto
+```
+
+This will start a broker on localhost listening on port 1883. For production use cases, MQTT would use TLS and access control, which is skipped here for simplicity.
+
+We separate the messages for the different vehicles by using the following topic scheme:
+
+```text
+vehicle/<VIN>/
+```
+
+Our example vehicle is assigned VIN 1. Let's listen to all messages to and from this vehicle by subscribing to the topic `vehicle/1/#`:
+
+```shell
+podman run --net=host docker.io/eclipse-mosquitto mosquitto_sub -h localhost -t "vehicle/1/#" -v
+```
+
+Keep this window open for the duration of this tutorial.
+
+## Fleet connector
+
+The fleet connector is a containerized workload managed by Ankaios. It will have two connections:
+
+1. MQTT connection to the cloud in order to receive messages for starting, stopping and updating workloads in the vehicle and to return the response.
+2. Connection to the Ankaios [control interface](../reference/control-interface.md) in order to execute the instructions to start, stop, and update workloads.
+
+The control interface is made available to each workload via named pipes (FIFO) using a protobuf IDL. In this tutorial, we will use the Ankaios SDK for Python, [ank-sdk-python](https://github.com/eclipse-ankaios/ank-sdk-python), which provides a convenient way to access the control interface.
+
+Let's take a look at the Fleet Connector implementation:
+
+```python
+from ankaios_sdk import Workload, Ankaios, WorkloadStateEnum, WorkloadSubStateEnum, AnkaiosLogLevel, Manifest, Request, CompleteState
+import paho.mqtt.client as mqtt
+import json
+import os
+import logging
+import sys
+
+logger = logging.getLogger("fleetconnector")
+stdout = logging.StreamHandler(stream=sys.stdout)
+stdout.setLevel(logging.INFO)
+logger.addHandler(stdout)
+logger.setLevel(logging.INFO)
+
+# Configuration for MQTT  broker and topics
+BROKER = os.environ.get('MQTT_BROKER_ADDR', 'localhost')
+PORT = int(os.environ.get('MQTT_BROKER_PORT', '1883'))
+VEHICLE_ID = os.environ.get('VIN')
+BASE_TOPIC = f"vehicle/{VEHICLE_ID}"
+
+# Create a new Ankaios object.
+# The connection to the control interface is automatically done at this step.
+ankaios = Ankaios()
+
+# Callback when the client receives a CONNACK response from the MQTT server
+def on_connect(client, userdata, flags, reason_code, properties):
+    client.subscribe(f"{BASE_TOPIC}/manifest/apply/req")
+    client.subscribe(f"{BASE_TOPIC}/manifest/delete/req")
+    client.subscribe(f"{BASE_TOPIC}/state/req")
+
+def convert_manifest_result_to_dict(result):
+    dict = { "added_workloads": [], "deleted_workloads": [] }
+    for a in result["added_workloads"]:
+        dict["added_workloads"].append(a.__dict__)
+    for d in result["deleted_workloads"]:
+        dict["deleted_workloads"].append(d.__dict__)
+    return dict
+
```

Comment:
We don't need this as the SDK is already providing a dict:
```suggestion
```

Reply:
I removed that.